integration/mediation-tests/pom.xml

```diff
@@ -31,19 +31,35 @@
     <name>WSO2 Enterprise Integrator - Mediation Integration Tests</name>
     <packaging>pom</packaging>
 
-    <modules>
-        <module>tests-common</module>
-        <module>service-samples</module>
-        <module>tests-transport</module>
-        <module>tests-service</module>
-        <module>tests-mediator-1</module>
-        <module>tests-mediator-2</module>
-        <module>tests-patches</module>
-        <module>tests-sample</module>
-        <module>tests-other</module>
-        <module>tests-ui</module>
-        <module>tests-platform</module>
-        <module>coverage-report</module>
-    </modules>
+
+    <profiles>
+        <profile>
+            <id>smoke-tests</id>
+            <modules>
+                <module>tests-common</module>
+                <module>tests-sample</module>
+            </modules>
+        </profile>
+        <profile>
+            <id>default</id>
```

Comment:
Shall we move the default profile to the top like in other POM files?

Reply:
done